### PR TITLE
x11-drivers/nvidia-drivers: Fix SRC_URI in 381.22

### PR DESCRIPTION
**Pull request by tkemmer**
Gentoo-Bug: https://bugs.gentoo.org/show_bug.cgi?id=618224

The ebuild for driver version 381.22 still uses the broken SRC_URI (that has been recently replaced in the other ebuilds via f41d9628eae446301363c2ec84fc9c495d6ff4b6). In addition, the URI for the ARM driver has changed.

See also:
https://devtalk.nvidia.com/default/topic/1007999/unix-graphics-announcements-and-news/linux-solaris-and-freebsd-driver-381-22/

### CODE REVIEW

File: x11-drivers/nvidia-drivers/nvidia-drivers-381.22.ebuild [hunk 1]
```diff
@@ -17,7 +17,7 @@ HOMEPAGE="http://www.nvidia.com/ http://www.nvidia.com/Download/Find.aspx"
 SRC_URI="
 	amd64-fbsd? ( ${NV_URI}FreeBSD-x86_64/${PV}/${AMD64_FBSD_NV_PACKAGE}.tar.gz )
 	amd64? ( ${NV_URI}Linux-x86_64/${PV}/${AMD64_NV_PACKAGE}.run )
-	arm? ( ${NV_URI}Linux-x86-ARM/${PV}/${ARM_NV_PACKAGE}.run )
+	arm? ( ${NV_URI}Linux-32bit-ARM/${PV}/${ARM_NV_PACKAGE}.run )
```
Comment (mgorny) [inline x11-drivers/nvidia-drivers/nvidia-drivers-381.22.ebuild]: Is a similar change needed in the remaining versions as well?

Comment (tkemmer) [inline x11-drivers/nvidia-drivers/nvidia-drivers-381.22.ebuild]: Hmm, good point! Actually, yes, all versions down to (including) 340.102 are affected. I will split this into separate commits then and fix the other versions as well. Do I need to file a separate bug report for this?

Comment (mgorny) [inline x11-drivers/nvidia-drivers/nvidia-drivers-381.22.ebuild]: Nah, I'll try to ping jer to look at this.

**Comment by mgorny** [comment]
It seems that he fixed one of the issues and removed the PR link without any comment. I wouldn't expect any polite behavior from him, so I think it'd be best if you filed a separate bug for the ARM URI fix. No point in submitting a pull request since he is just going to ignore it.

**Comment by tkemmer** [comment]
I see, can't be helped then. I will try a separate bug report for the broken ARM URIs... Thanks anyway! :)

**Pull request closed without merging**